### PR TITLE
Add AllowMediaTranscription description

### PR DESCRIPTION
**Pull request by mskaai**
There is an updated value that allow for admins to manage transcription generation for all videos in ODSP. This will control the initial on demand feature allowing owners to request a transcript on demand as well as the longer term automatic transcription feature.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -203,6 +204,28 @@ This example enable the option to search for existing guest users at Tenant Leve
...
 
+
+### -AllowMediaTranscription
+
+When the feature is enabled, videos can have transcripts generated on demand or generated automatically in certain scenarios. This is the default because the policy is default on. If a video owner decides they don’t want the transcript, they can always hide or delete it from that video. 
+
+The valid values are:  
+
+- True (default) - Videos can have transcripts generated on demand.
+- False - Videos cannot have transcripts generated by Stream in any scenario. Note there are other scenarios where transcript could still exist.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:215]:
```suggestion
- False - Videos cannot have transcripts generated by Stream in any scenario. Note that there are other scenarios where transcripts could still exist.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mskaai, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by get-itips** [comment]
Hello @mskaai 
Same question as https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/108#issuecomment-995946715

**Comment by yogkumgit** [comment]
@mskaai Please update

**Comment by mskaai** [comment]
> Collaborator

Hi, it should be published in version 16.0.21909.12000. This is also the old PR, I actually submitted an updated PR that went through I think.

**Comment by get-itips** [comment]
> > Collaborator
> 
> Hi, it should be published in version 16.0.21909.12000. This is also the old PR, I actually submitted an updated PR that went through I think.

Do you mean the PR 108? It is for Set-SPOSite and this is for Set-SPOTenant, please confirm if this PR is still valid, thanks

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd

**Pull request merged**